services/storage/src/simcore_service_storage/api/rpc/_simcore_s3.py

```diff
@@ -38,8 +44,14 @@ async def start_export_data(
     task_manager: TaskManager,
     job_filter: AsyncJobFilter,
     paths_to_export: list[PathToExport],
+    export_as: Literal["path", "download_link"],
```

Comment:
1. would using an enum not make the whole code a bit easier?
2. how is this function used at the moment? from my point of view the UI also downloads these zip files somehow right? how does this work? how does it create the download link in this case?